src/main/java/com/meta/cp4m/message/Message.java

```diff
@@ -13,13 +13,17 @@
 
 public interface Message {
 
-  static Identifier threadId(Identifier id1, Identifier id2) {
+  public static Identifier threadId(Identifier id1, Identifier id2) {
     if (id1.compareTo(id2) <= 0) {
       return Identifier.from(id1.toString() + '|' + id2);
     }
     return Identifier.from(id2.toString() + '|' + id1);
   }
```

Comment:
can we make this package-private again now?

Reply:
Addressed this!